Formula/opencv.rb

```diff
@@ -45,9 +45,10 @@ def install
     # Reset PYTHONPATH, workaround for https://github.com/Homebrew/homebrew-science/pull/4885
     ENV.delete("PYTHONPATH")
 
-    py3_config = `python3-config --configdir`.chomp
-    py3_include = `python3 -c "import distutils.sysconfig as s; print(s.get_python_inc())"`.chomp
-    py3_version = Language::Python.major_minor_version "python3"
+    xy = Language::Python.major_minor_version "python3"
```

Comment:
Looks like Python 3.7 is being brought in somewhere along the dependency tree as this returns "3.7" on the CI.

Either do:

```suggestion
    xy = Language::Python.major_minor_version Formula["python@3.8"].opt_bin/"python3"
```

or remove it all together since `PYTHON3_LIBRARY` should now be able to be deduced from `PYTHON3_EXECUTABLE` after the CMake changes.

Reply:
I removed it and it worked.